docker/root/dbase.config.php

```diff
@@ -5,10 +5,10 @@
 const HLEB_PARAMETERS_FOR_DB = [
     'mariadb' => [
         'mysql:host=db',
-        'port=3306',
-        'dbname=hleb',
+        'port=MYSQL_DATABASE_PORT',
```

Comment:
```suggestion
        'port=3306',
```
Поскольку переменной "MYSQL_DATABASE_PORT" в файле `.env` не будет, эту строку также не изменяем

Reply:
Разработчик через эту переменную может подключится не только к текущей базе данных, но и к любой возможной (допустим есть test и prod в другом контейнере).

Reply:
Понял. Да, это разумно, надо будет обдумать